docs/reference/ccr/getting-started.asciidoc

```diff
@@ -74,17 +86,16 @@ cluster privilege.
 
 [[ccr-getting-started-remote-cluster]]
 ==== Connect to a remote cluster
-Connect your local cluster to a
-<<modules-remote-clusters,remote cluster>> to begin using cross-cluster
-replication.
+Connect your local cluster (`ClusterA`) to a remote cluster (`ClusterB`) to
+begin using {ccr}.
 
-To configure a {kibana-ref}/working-remote-clusters.html[remote cluster],
-access {kib} and go to
+To configure a remote cluster, access {kib} and go to
```

Comment:
Instead of "access Kibana", I'd probably just say:

To configure a remote cluster from Stack Management in Kibana:

. Select *Remote Clusters* from the side navigation. 
. Specify the IP address or host name of the remote...